hooks/post/30-upgrade.rb

```diff
@@ -66,6 +66,15 @@ def remove_gutterball
   end
 end
 
+def remove_event_queue
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
```

Comment:
How sure are we these paths, ports etc are correct? Would it make sense to use values from the answers file? Also wondering if this should also use the Kafo execute helper.